webpack/assets/javascripts/react_app/components/Pagination/PaginationHelper.js

```diff
@@ -0,0 +1,21 @@
+import URI from 'urijs';
+
+const uri = new URI(window.location.href);
```

Comment:
I afraid the `uri` reviving value when the page load and won't get recalculate later.
Could you use:
```js
export const getUri => new URI(window.location.href);
```

Then you can also use it from your component.